docs/source/release_notes/release-notes.rst

```diff
@@ -42,7 +42,11 @@ develop
          - Change
 
     *    - |fixed|
-         - Multinode Cassandra: ``getAllTimestamps`` and ``getRangeOfTimestamps`` now work as long as quorum for reading is achieved.
+         - Multinode Cassandra: ``createTable(s)`` and ``dropTable(s)`` now throw an IllegalStateException if a node is down in a multinode Cassandra cluster.
```

Comment:
This is breaking too - in that we've changed the exception type from `PalantirRuntimeException`. I think we should call that out explicitly.